### ISSUE TITLE
[Checkbox]: WCAG and other issues

### ISSUE THREAD

**Issue by ivarkallejarv** [TEHIK-EE/tedi-design-system#807]
## Problem

More of a discussion probably:
- Create a checkbox with multiple options, one option should be disabled and should also have a tooltip inside it. Should the tooltip inside a disabled Checkbox's content be focusable? 

"Rule: ARIA hidden element must not be focusable or contain focusable elements https://dequeuniversity.com/rules/axe/4.10/aria-hidden-focus"

![Image](https://github.com/user-attachments/assets/39802b23-b1aa-4ac6-a8d2-09e2a5d6af1c)

- Create a list of checkboxes, add an option to select/deselect all. The given role is probably wrong.

"Rule: ARIA attributes must be used as specified for the element's role https://dequeuniversity.com/rules/axe/4.10/aria-conditional-attr"

![Image](https://github.com/user-attachments/assets/fec8c953-0321-47f9-9ba0-be5cbe6c29ac)

![Image](https://github.com/user-attachments/assets/4ce1d088-d4d9-4469-87a1-a12269b147ec)

**Comment by tobre6** [comment]
1. We can't make decision regarding WCAG. It depends on the usecase. On some cases you should be able to see tooltip even if the checkbox is disabled (then it should be focusable) other cases tooltip should be disabled as well. Regarding WCAG both ways are argumentable.
2. Can't see issue here. Works as intended. Screen readers are reading correctly. Tested with JAWS and Voicover.
